src/scss/colors.scss

```diff
@@ -0,0 +1,38 @@
+@use "sass:color";
+
+$colors-enum: (
+  primary: #7c46fb,
+);
+$shades-enum: (5, 10, 14, 20, 50);
+$alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
+
+@function hexToRGB($hex) {
+  @return red($hex), green($hex), blue($hex);
+}
+
+:root {
+  --primary-color-deprecated: #f6e652;
+
+  --primary-content--under: #000000;
+  --primary-content--under-rgb: #{hexToRGB(#000000)};
+  --primary-content--over: #ffffff;
+  --primary-content--over-rgb: #{hexToRGB(#ffffff)};
+  --primary-accent--over: #ffffff;
+
+  @each $colorName, $color in $colors-enum {
+    @each $shade in $shades-enum {
+      --#{$colorName}--darker-#{$shade}pp: #{darken($color, $shade * 1%)};
+      --#{$colorName}--lighter-#{$shade}pp: #{lighten($color, $shade * 1%)};
+    }
+
+    --#{$colorName}-accent--under: #{$color};
+    --#{$colorName}-accent--under-rbg: #{hexToRGB($color)};
```

Comment:
Why is this needed as a prop variable on :root ? The `-rbg` version can just be a constant before `@each` since in the CSS there will already be `--#{$colorName}-accent--under` for use, no need of duplicates there.

Reply:
fixed in 8e20515ab04bf4df178d893dadb185d0e7b6126e